modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -296,6 +296,14 @@ definition {
 		}
 	}
 
+	macro assertCannotDeleteAllAssignments {
+		Click(locator1 = "ProcessBuilderKaleoDesignerReact#SELECTED_ASSIGNMENTS_VALUE");
+
+		AssertNotEditable(
+			key_text = "trash",
+			locator1 = "Icon#ANY_DISABLED");
+	}
+
```

Comment:
@patriciajeanperez I think that way too, I just did it because I thought it was a workflow test pattern.We have a lot of asserts that should be at the testcase level. That would be a good refactoring.